libtsuba/src/WriteGroup.cpp

```diff
@@ -66,19 +40,19 @@ WriteGroup::AddOp(
   }
   if (accounted_size > 0) {
     while (outstanding_size_ + accounted_size > kMaxOutstandingSize) {
-      if (!Drain()) {
+      if (!async_op_group_.FinishOne()) {
```

Comment:
I can't imagine it's a real issue, but won't the FIFO ordering of `AsyncOpGroup::FinishOne()` be suboptimal here?

Reply:
I need to run the callbacks in order so that properties go into tables in the order that they did before. I was also hoping that  it's not a real issue, but you're right that we could relax this here to be `FinishAny`.

Reply:
I forgot about the callback ordering guarantee.

Reply:
I wonder if we should use arrow Tables at all for properties.  Why not just use columns?

Reply:
We could talk about that; probably more churn than it's worth in this PR.